test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -60,6 +60,9 @@
     ("aten::linalg_outer", datetime.date(2020, 8, 30)),
     ("aten::linalg_outer.out", datetime.date(2020, 8, 30)),
     ("aten::_compute_linear_combination", datetime.date(2020, 9, 1)),
+    # TODO before land: make the matching logic in this test more generic
+    #   to allow matching by function arguments
+    ("getstate", datetime.date(2020, 9, 1)),
```

Comment:
why do you need it? and why is it named without the prefix?

Reply:
> why do you need it?

Afaik changing the return type of `__getstate__` will break this test as written, because it checks for function signatures.  We are saying (and I'd love feedback if this is incorrect) that this breakage is OK because all the user flows in Quantization callsits will still work via our patch, and we are OK with breaking non-Quantization code which depended on the old return type (we don't know of any real instances of this).

> why is it named without the prefix?

should be fixed after the TODO is implemented, we need to update this test a bit to also allow matching on arguments (currently it just matches on `__getstate__` which would match too many things).

Reply:
makes sense, but I don't think you need matching on arguments. We return the name of the same as a non-qualified name right now and it doesn't make much sense. We could just change code here (https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/custom_class.cpp#L49) to return `fn->qualname` as the name for the schema

or as you pointed out, we could post process it in python and just prepend `.arguments[0].type.name()` to the name (assuming that `self` is always the first argument)